driver-core/src/main/com/mongodb/connection/netty/NettyStream.java

```diff
@@ -385,6 +384,39 @@ public T get() throws IOException {
         }
     }
 
+    private class OpenChannelFutureListener implements ChannelFutureListener {
+        private final Queue<SocketAddress> socketAddressQueue;
+        private final ChannelFuture chlFuture;
+        private final AsyncCompletionHandler<Void> handler;
+
+        OpenChannelFutureListener(final Queue<SocketAddress> socketAddressQueue, final ChannelFuture chlFuture,
+                                  final AsyncCompletionHandler<Void> handler) {
+            this.socketAddressQueue = socketAddressQueue;
+            this.chlFuture = chlFuture;
```

Comment:
Note: Its good practise to follow consistent naming schemes.  Previously `channelFuture` was used and here its `chlFuture`. Using the same name will lower the cognitive load of the code.

Reply:
Done